packages/parser-typescript-config/README.md

```diff
@@ -0,0 +1,61 @@
+# Parser typescript-config (`@sonarwhal/parser-typescript-config`)
+
+The `typescript-config` parser allows the user to analyze the
+TypeScript configuration in their projects.
+
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @sonarwhal/parser-typescript-config
+```
+
+If you want to install it globally, add the parameter `-g`.
+If you want to install it as a dev dependency, add the parameter `--save-dev`
+
+And then activate it via the `.sonarwhalrc` configuration file:
+
+```json
+{
+    ...
+    "parsers": ["typescript-config"],
+    "rules": {
+        ...
+    },
+    ...
+}
+```
+
```

Comment:
We should explain that we use the name of the file to detect the TypeScript configuration and what are the limitations (`tsconfig.json`, `tsconfig.development.json`, we don't detect arguments, etc.)